woocommerce/woocommerce-gutenberg-products-block
# Products by Rating: Add Filter by Rating to the Feature Plugin

This PR moves the `Filter by Rating` block from `isExperimentalBuild` to `isFeaturePluginBuild`.

#### Other Checks

- [x] This PR adds a feature flag & I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
- [x] This PR removes an experimental interfaces and I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).

### Testing
1. Check out `trunk`
2. Build and test using `npm run build WOOCOMMERCE_BLOCKS_PHASE=2` - check that the Ratings filter is not available
3. Check out this branch
4. Build and test using `npm run build WOOCOMMERCE_BLOCKS_PHASE=2` - check that the Ratings filter **is available**

#### Automated Tests


### WooCommerce Visibility




### Changelog

> Filter by Ratings: Add the Filter by Rating block to the feature plugin build.
